swagger_server/messaging/topic_queue_consumer.py

```diff
@@ -90,8 +90,14 @@ def handle_mq_msg(self, msg_body):
                 self.message_id += 1
                 self.logger.info("Sending connection info to Kytos.")
                 # Uncomment lines below to send connection info to Kytos
-                r = requests.post(str(KYTOS_URL), json=msg_json)
-                self.logger.info("Status code:" + str(r.status_code))
+                try:
+                    r = requests.post(str(KYTOS_URL), json=msg_json)
+                    self.logger.info("Status code:" + str(r.status_code))
+                except ConnectionRefusedError:
+                    self.logger.info(
+                        "Cannot connect to Kytos, "
+                        "check your configuration and make sure kytos is running."
+                    )
```

Comment:
I guess other errors are possible too -- name lookup failure, routing failure, and so on?  Can we "broaden" the exception catching like below?

```python
except Exception as e:
    self.logger.info(f"Error on POST to {KYTOS_URL}: {e}")
```

Or maybe catch `ConnectionError`, not just the subclass.

Reply:
Also log level here should be `self.logger.error()`, not `info`, I think.